example/src/main/java/com/ryanharter/auto/value/gson/example/PersonTypeAdapterFactory.java

```diff
@@ -0,0 +1,18 @@
+package com.ryanharter.auto.value.gson.example;
+
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+import com.google.gson.TypeAdapterFactory;
+import com.google.gson.reflect.TypeToken;
+
+public class PersonTypeAdapterFactory implements TypeAdapterFactory {
```

Comment:
GOt it

On Wed, Apr 20, 2016 at 11:10 AM Piasy <notifications@github.com> wrote:

> In
> example/src/main/java/com/ryanharter/auto/value/gson/example/PersonTypeAdapterFactory.java
> <https://github.com/rharter/auto-value-gson/pull/30#discussion_r60426264>:
>
> > @@ -0,0 +1,18 @@
> > +package com.ryanharter.auto.value.gson.example;
> > +
> > +import com.google.gson.Gson;
> > +import com.google.gson.TypeAdapter;
> > +import com.google.gson.TypeAdapterFactory;
> > +import com.google.gson.reflect.TypeToken;
> > +
> > +public class PersonTypeAdapterFactory implements TypeAdapterFactory {
>
> But we must have a gson instance to create the TypeAdapter instance :(
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/rharter/auto-value-gson/pull/30/files/e2d8ec9e7411524859dd3b2cfe98a76e6f3ce9e5#r60426264>
>